src/components/Chat/ChatMessage.tsx

```diff
@@ -104,10 +99,10 @@ class ChatMessage extends UIComponent<Extendable<ChatMessageProps>, any> {
   static propTypes = {
     ...commonUIComponentPropTypes,
     ...childrenComponentPropTypes,
-    ...contentComponentPropsTypes,
     accessibility: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
     author: customPropTypes.itemShorthand,
     avatar: customPropTypes.itemShorthand,
+    content: customPropTypes.itemShorthand,
```

Comment:
just to be sure - this is done because of `content` prop type (`contentShorthand`) that doesn't fit here?

Reply:
yes